scalardb-server-sample/build.gradle

```diff
@@ -11,18 +11,9 @@ repositories {
 }
 
 dependencies {
-    implementation group: 'com.scalar-labs', name: 'scalardb', version: '3.4.1'
-    implementation group: 'info.picocli', name: 'picocli', version: '4.6.2'
-    implementation group: 'org.slf4j', name: 'slf4j-simple', version: '1.7.32'
-
-    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.8.2'
-    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.8.2'
+    implementation 'com.scalar-labs:scalardb:3.6.0'
 }
 
 application {
-    mainClass = 'example.ElectronicMoneyMain'
-}
-
-test {
-    useJUnitPlatform()
+    mainClass = !project.hasProperty("storage") ? 'sample.ElectronicMoney' : 'sample.ElectronicMoneyWithStorage'
```

Comment:
If we run it with the `storage` property (`-Pstorage`), Storage mode is executed.